styles/pages/statistics.module.scss

```diff
@@ -0,0 +1,16 @@
+.stats {
+	display: flex;
+	align-content: flex-start;
```

Comment:
this doesn't do anything as of right now

Reply:
noted